Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/ShellTitleView.cs

```diff
@@ -15,6 +16,7 @@ namespace Xamarin.Forms.Controls.Issues
 #if UITEST
 	[NUnit.Framework.Category(UITestCategories.Shell)]
 	[NUnit.Framework.Category(UITestCategories.TitleView)]
+	[NUnit.Framework.Category(UITestCategories.UwpIgnore)]
```

Comment:
@hartez not sure what exactly you were looking at / working on with UWP TitleView, but just wanted to point this out in case it somehow ends up conflicting or relating!

Reply:
Based on the chat from our last standup I'm going to assume it doesn't relate :-)